nixos/modules/services/networking/osmocom/osmo-hlr.nix

```diff
@@ -0,0 +1,92 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.osmo-hlr;
+in
+{
+  options = {
+    services.osmo-hlr = {
+      enable = lib.mkEnableOption (lib.mdDoc "OsmoHLR");
+      package = lib.mkPackageOptionMD pkgs "osmo-hlr" { };
+      flags = lib.mkOption {
+        default = [ ];
+        type = lib.types.listOf lib.types.str;
+        example = [ "--timestamp" ];
+        description = lib.mdDoc ''
+          Flags to append to the program call
+          '';
+      };
+      settings = lib.mkOption {
```

Comment:
`settings` is usually used for RFC42 compliant options
Prefer `config` or `configFile` and use the appropriate `mkDefault` so that it's easier for users to manipulate